visualizations/frontend/other/TBigNumber.vue

```diff
@@ -60,6 +63,7 @@
       </t-tooltip>
       <div v-else class="opacity-80">No change</div>
     </div>
+    <slot name="bigNumFooter"></slot>
```

Comment:
Because this is already inside the big number, it is probably better to just call this slot `footer` instead of `bigNumFooter`.

@icoconutgirl